servers/physics_3d/godot_collision_solver_3d_sat.cpp

```diff
@@ -1962,7 +1962,7 @@ static void _collision_cylinder_face(const GodotShape3D *p_a, const Transform3D
 
 	// Points of B, cylinder lateral surface.
 	for (int i = 0; i < 3; i++) {
-		const Vector3 &point = vertex[i];
+		const Vector3 &point = vertex[i] - p_transform_a.origin;
```

Comment:
```suggestion
		const Vector3 point = vertex[i] - p_transform_a.origin;
```
Can't take a reference to an expression safely

Reply:
Good point! I fixed the commit